File: C#/begginer/1015.cs
using System;

class URI {

    static void Main(string[] args) { 

        string[] input1 = Console.ReadLine().Split(' ');
        double[] p1 = { double.Parse(input1[0]), double.Parse(input1[1]) };

        string[] input2 = Console.ReadLine().Split(' ');
        double[] p2 = { double.Parse(input2[0]), double.Parse(input2[1]) };

        Console.WriteLine($"{Math.Sqrt(Math.Pow(p2[0] - p1[0], 2) + Math.Pow(p2[1] - p1[1], 2)):F4}");
    }
}

File: Java/begginer/1015.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        
        double[][] points = new double[2][2];
        
        for(int i = 0; i < 2; i++) {
            String[] input = scanner.nextLine().split(" ");
            for(int j = 0; j < 2; j++) {
                points[i][j] = Double.valueOf(input[j]).doubleValue();
            }
        }
        
        System.out.printf("%.4f\n", Math.sqrt(Math.pow(points[1][0] - points[0][0], 2) + Math.pow(points[1][1] - points[0][1], 2)));
        
    }
 
}
